### PR TITLE
Add mocks matching algo, custom proxy, travis integration

### PR DESCRIPTION
**Pull request by loopDelicious**
- add mocks matching algo page
- add custom proxy page
- add travis ci integration
- update header to mirror marketing website
- copy fix, no parameter tables are generated in documentation
- UI fix, clear search stats after clearing search box
- correction in using environments with collection runs, variables ARE persisted by default, new collection and env JSON files uploaded

@godfrzero can you please review? the last point is a complete change from what was documented previously - `Persist Variables` is checked as default, correct? not sure who on the App team to verify.

### CODE REVIEW

File: _includes/_navigation.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
     <ul class="pm-sidebar-main-nav">
-        <li><a href="{{site.pm.root}}/">Home</a></li>
-        <li><a href="{{site.pm.root}}/products">Products</a></li>
-        <li><a href="{{site.pm.root}}/pricing">Pricing</a></li>
-        <li><a href="{{site.pm.root}}/docs/">Documentation</a></li>
-        <li><a href="{{site.pm.root}}/community">Community</a></li>
+        <li><a href="{{site.pm.root}}/" class="<%- current.path[0] == 'index' ? 'active' : '' %>">Home</a></li>
+        <li><a href="{{site.pm.root}}/products" class="<%- current.path[0] == 'products' ? 'active' : '' %>">Products</a></li>
+        <li><a href="{{site.pm.root}}/pricing" class="<%- current.source == 'pricing' ? 'active' : '' %>">Pricing</a></li>
+        <li><a href="{{site.pm.root}}/docs/" class="<%- current.path[0] == 'docs' ? 'active' : '' %>">Documentation</a></li>
```
Comment (godfrzero) [inline _includes/_navigation.html]: Won't `docs` always be the active tab?

File: _posts/01_postman/02_sending_api_requests/2017-05-04-proxy.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+In the first field under **Proxy Server**, enter the **host** or **IP address** (without protocol) of the proxy server. In the second field under **Proxy Server**, enter the **port** of the proxy server.
+
+[![custom proxy](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/proxy_custom.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/proxy_custom.png)
+
+### Using system proxy
+
+This is enabled by default. Any request made through Postman will go through the system proxy.
+
+You can turn on and off this setting using the toggle switch. When turned off, all the requests are made directly.
```
Comment (godfrzero) [inline _posts/01_postman/02_sending_api_requests/2017-05-04-proxy.md]: First sentence sounds a bit weird. Should probably be "You can turn this setting on and off using the toggle switch".

File: _posts/01_postman/02_sending_api_requests/2017-05-04-proxy.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+[![system proxy](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/proxy_system.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/proxy_system.png)
+
+**NOTE:** If the **System Proxy** and the **Custom Proxy** are both turned on, then the **Custom Proxy** will take precedence.
+
+In the latest Canary builds available for download on OSX ([x64](https://dl.pstmn.io/download/channel/canary/osx_64)) / Windows ([x86](https://dl.pstmn.io/download/channel/canary/windows_32) or [x64](https://dl.pstmn.io/download/channel/canary/windows_64)) / Linux ([x86](https://dl.pstmn.io/download/channel/canary/linux_32) or [x64](https://dl.pstmn.io/download/channel/canary/linux_64)) / [Chrome](https://chrome.google.com/webstore/detail/postman/loddepljkabcghihdkbkfknabkinanff), you can use these proxies.
```
Comment (godfrzero) [inline _posts/01_postman/02_sending_api_requests/2017-05-04-proxy.md]: This sentence would sound more active if "you can use these proxies" was at the beginning.

File: _posts/01_postman/06_collection_runs/2017-05-04-integration_with_travis.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  [![tree view yml](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/travis_tree_yml.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/travis_tree_yml.png)
+  In the `.travis.yml` file, add a command to `install` Newman in the CI environment, and then add a `script` telling Newman to run the Postman tests (which I've placed in the `tests` directory). Since Travis CI doesn’t know where Newman is located, let's update the `PATH`. In this node.js example, the `newman` tool is located in my `.bin` directory which is located in my `node_modules` directory.
+  
+Now, my `.travis.yml` file looks like this for this node.js example:
+
+<script src="https://gist.github.com/loopDelicious/c8c43557e0229e5c20341913a42cf528.js"></script>
```
Comment (godfrzero) [inline _posts/01_postman/06_collection_runs/2017-05-04-integration_with_travis.md]: Might want to wrap this within a code block. ```

**Comment by loopDelicious** [comment]
@godfrzero made the requested changes. please review.

looks like persisting variables is as you suggested: default is checked first time you open the runner. updated the language accordingly.

**Pull request merged**